### PR TITLE
Feature/parking map

### PR DESCRIPTION
**Pull request by addelian**
My Prettier seems to have done a number on the formatting on a lot of these files - apologies! The only things that were actually changed were the parking lat/long in model, the getCurrentParkingLocation function in the edit modal.ts, the showParkingMap function in the location-camp.ts, and all of the related html on the pages themselves.

### CODE REVIEW

File: src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html [hunk 1]
```diff
@@ -1,79 +1,209 @@
...
+          <label for="parking-latitude" class="col-6 col-form-label"
+            >Parking Latitude:
+          </label>
+          <div class="col-3">
+            <button
+              class="btn btn-primary"
+              style="float: right"
+              (click)="getCurrentParkingLocation()"
+            >
+              Get Current Location
```
Comment (wilsont3) [inline src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html:105]: Should we somehow indicate that this button changes the parking location coordinates?

**Review by wilsont3 — COMMENTED** [review]
You did a great job with this!  What are your thoughts on if we should indicate what fields you are updating with the Get Current Location buttons?

File: src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html (hunk 1 above)
Comment (addelian) [reply src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html:105]: @wilsont3 that makes perfect sense to me. I'll update the wording!

Comment (addelian) [reply src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html:105]: Screen Shot 2022-01-08 at 11 04 42 PM

@wilsont3 Thoughts on this? I initially updated just the wording for parking, but then it felt strange that the original functionality was vague.

Comment (wilsont3) [reply src/app/location-camp/camp-edit-modal/camp-edit-modal.component.html:105]: That looks great.  I was kind of thinking the same thing.

**Comment by addelian** [comment]
@wilsont3 I updated the labels as well as the placeholders in both places. Still functioning as expected. Let me know if you'd like to see any other changes!

**Review by wilsont3 — APPROVED** [review]
This looks great!  Thanks for doing this @addelian

**Pull request merged**